Releases/Boston/README.md

```diff
@@ -0,0 +1,16 @@
+# Skuid Boston
+
+Resources to guide you through the features that come with Skuid Boston release (v13.X), including a demo page and an accompanying design system. 
+
+## About Skuid Boston
+Link to [Skuid Boston's Release Notes](https://docs.skuid.com/latest/en/release-notes.html).
```

Comment:
Let's link this to the version-specific release notes so this link stays accurate in the future.
The `v13.0.4` docs are publishing now, so this link should be live by around 3:40 PM EST

Also adding a line break between the header and the section text.

```suggestion

Link to [Skuid Boston's Release Notes](https://docs.skuid.com/v13.0.4/en/release-notes.html).
```